test/functional/fixtures/api/es-next/hybrid-function/testcafe-fixtures/hybrid-fn-test.js

```diff
@@ -56,3 +56,37 @@ test('Generator in Hybrid', async () => {
         yield 1;
     });
 });
+
+test('Bind Hybrid function', async t => {
+    const fs               = require('fs');
+    const boundGetLocation = getLocation.bindTestRun(t);
+
+    // NOTE: binding should not modify original Hybrid
+    // function bound test run, so it should throw.
```

Comment:
Nope, but my version was incorrect as well. The meaning is that binding doesn't change original function's binding, but creates new bound function instead.